ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -513,6 +513,26 @@ AHU_DFSS_DSP_DX2ZC_ECONZ_EFSS_HTZC:
   - EFSS
   - HTZC
 
+AHU_SFSS_SFVSC_EFSS_CHWSC_RTM_FDPM2X_CO2EFVSC_SSPC2X_CO2M6X_ZTM_ZHM_OAFC_VOCM_PMM:
+  description: "Multi zone AHU."
+  is_canonical: true
+  implements:
+  - AHU
+  - SFSS
+  - EFSS
+  - SFVSC
+  - CHWSC
+  - RTM
+  - FDPM2X
+  - CO2EFVSC
+  - SSPC2X
+  - CO2M6X
+  - ZTM
```

Comment:
Which zone then? if we have 6 zones, i would expect something like a ZTM6X abstract type, same for ZHM6X

Reply:
Sorry, removed ZTM, ZHM as these were not part of the AHU.